utils/firestore.js

```diff
@@ -2125,6 +2125,53 @@ const updateUserPhoneSigninMethod = async (phone, uid) => {
     }
 }
 
+/**
+ * queryDailyReportParticipants
+ * @param {}
+ * returns participants that are no more than 2 days old from the time of check in date/time
+ * Using promises to resolve array of objects
+ * object containes essential information Collection Location (951355211), Connect ID,
+ * F+L Name (996038075 & 399159511), Check-In (840048338), Collection ID (820476880),
+ * Collection Finalized(556788178), Check-Out (343048998)
+ * Queried from Participants & Biospecimen table
+ */
+
+const queryDailyReportParticipants = async () => {
+    const twoDaysAgo = new Date(new Date().getTime() - (2 * 24 * 60 * 60 * 1000)).toISOString();
+    let query = db.collection('participants');
+    const snapshot = await query.where('331584571.266600170.135591601', '==', 353358909).where('331584571.266600170.840048338', '<', twoDaysAgo).get();
+    if (snapshot.size !== 0) {
+        const promises = snapshot.docs.map(async (document) => {
+            return processQueryDailyReportParticipants(document);
+        });
+        return Promise.all(promises).then((results) => {
+            return results.filter((result) => result !== undefined);
```

Comment:
Question: Do we have to worry about null or falsy values here?
`return results.filter((result) => result != null);` //filters null and undefined
`return results.filter((result) => result);` //filters anything falsy

Reply:
i dont think so